### DISCUSSION TITLE
Vue3 support

### DISCUSSION THREAD

**Discussion by prem4bio** [appbaseio/reactivesearch#1938] [Q&A]
Dear developers,

Can any of the contributors please add support for Vue3? I'm migrating the existing project to Vue3 and found that reactivesearch is not compatible with vue3. If there is any workaround available, please let me know.

Thank you.

**Comment by siddharthlatest** [comment]
This should be happening soon! Some packages reactivesearch-vue depends on haven't yet been ported or have issues working with Vue3. I would say in ~3 months or so, we should be there.

**Comment by lacorde** [comment]
Hi @siddharthlatest (& @bietkul), any news on this ?
Thank you for the great work !

**Reply by siddharthlatest** [reply]
@lacorde We should be able to get to Vue 3 support very soon (`~1mo`). One of the dependencies we relied on wasn't Vue3 compatible -`vue-google-maps`, and I see a fork of it that now supports Vue 3.

**Reply by lacorde** [reply]
Awesome, really looking forward to it !
